website/sidebars.json

```diff
@@ -56,6 +56,7 @@
       "highlight-untyped"
     ],
     "Experimental Features": [
+      "unsupported",
```

Comment:
Putting this here feels a little weird.  WDYT about putting it under "Troubleshooting" instead?